modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm

```diff
@@ -98,6 +98,13 @@
 /datum/armament_entry/company_import/nri_surplus/firearm
 	subcategory = "Firearms"
 
+/datum/armament_entry/company_import/nri_surplus/firearm/makarov
+	item_type = /obj/item/gun/ballistic/automatic/pistol/makarov
```

Comment:
and this badboy does a fair bit of damage, so should be questionably marginally more expensive then the shotgun revolver right below it, also, because it fits in pocket, maybe put into the 'illegal' options aswell

Reply:
Our 10mm does 24dmg with its reduction. So it's on par with the mini e-gun, both in damage and pocketability. Would that still warrant the move?

Reply:
Actually, I'm looking at it, and this is a bit excessive, yeah. I don't know where you get the 24 from, but from what I could tell, each bullet from this bad boy is 40 damage.

Reply:
You're right the 10mm round is normally 40dmg, and that would be wayyy overpowered for this price point. This particular makarov has a 0.6 projectile damage multiplier, which brings it down to 24. 
![makarovDefine](https://github.com/Skyrat-SS13/Skyrat-tg/assets/107971606/8ea4c0b2-0783-4fc1-8a25-5b83bc53b39c)

Confirmed the damage in a quick test.
![makarovShot](https://github.com/Skyrat-SS13/Skyrat-tg/assets/107971606/a5061c04-381f-47e3-a0eb-c4645b6531c1)